gradle/render-javadoc.gradle

```diff
@@ -151,6 +151,18 @@ class RenderJavadocTask extends DefaultTask {
 
   @Input
   def solrDocUrl = project.propertyOrDefault('solr.javadoc.url', "https://lucene.apache.org/solr/${project.baseVersion.replace(".", "_")}")
+  
+  /** Utility method to recursively collect all tasks with same name like this one that we depend on */
+  private Set findPreviousRenderTasks() {
```

Comment:
findRenderTasksInDependencies maybe?

Reply:
Yeah, much better! This name was puzzling me. I'll change!